tests/test_plugin.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+from poetry_partifact_plugin.plugin import PartifactPlugin
+
+
+def test_object_initialization():
+    """Unsure of the best way to test a poetry plugin..."""
```

Comment:
nit: Test docstrings typically describe what condition the test is evaluating. For example, this could read:
```py
"""Should correctly initialize the poetry plugin class."""
```